src/adapters/v00/marketplace_adapter.js

```diff
@@ -1,3 +1,5 @@
+const OFFER_STATUS = ['error','created', 'accepted', 'disputed', 'finalized', 'buyerReviewed']
```

Comment:
Could you also make those changes in v01/marketplace_adapter.js ?

Reply:
There's only one adapter file, both versions share it.